src/unix/fs.c

```diff
@@ -49,6 +49,7 @@
     defined(__FreeBSD__)    ||                                            \
     defined(__OpenBSD__)    ||                                            \
     defined(__NetBSD__)
+# include <sys/syslimits.h> /* PATH_MAX, IOV_MAX */
```

Comment:
hmmm... so changing it to limits works

--> https://ci.nodejs.org/job/node-test-commit-freebsd/2195/

So why does every system but bsd have the macros here?